src/dnstest_definitions.erl

```diff
@@ -838,6 +838,145 @@ erldns_dnssec_definitions() ->
             }
         }},
 
+        % Verify NXNAME query
+        % https://www.ietf.org/archive/id/draft-ietf-dnsop-compact-denial-of-existence-07.html#section-3.5
+        {nxname_query, #{
+            question => {"nxname.minimal-dnssec.com", 128},
```

Comment:
Don't we have a macro value for the `128` magic number? 🤔

Reply:
It needs to be added to dns_erlang.

Reply:
Updated in [bf3c50a](https://github.com/dnsimple/dnstest/pull/22/commits/bf3c50a7efcf04ee7b1089a23e554a2f10cf88ea)